src/main/java/core/basesyntax/exceptions/LoginAlreadyExistsException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exceptions;
+
+public class LoginAlreadyExistsException extends RuntimeException {
```

Comment:
I think it's better to create a single exception class like "BadValidationException" and just write the correct message where you want to throw the exception